src/controllers/auth.js

```diff
@@ -1,6 +1,6 @@
-const utils = require('../utils/utils')
+const Utils = require('../utils/utils')
 
-const users = [
+const Users = [
```

Comment:
```suggestion
const USERS = [
```

I meant this ^ 
Refer to: https://google.github.io/styleguide/jsguide.html#naming-constant-names